docs/installation.md

```diff
@@ -0,0 +1,170 @@
+# Installation
+
+Currently we support following ways, choose any of your comfort: 
+
+- [Power Shell](#power-shell) - easiest way
+- [Git Commands](#git-commands) - default way + adding manual settings
+- [Pyenv-win zip](#pyenv-win-zip) - manual installation
+- [Python pip](#python-pip) - for existing users
+- [Chocolatey](#Chocolatey)
+- [How to use 32-train](#how-to-use-32-train)  
+   - [check announcements](../README.md#Announcements)
+
+Hurray! When you are done here are steps to [Validate](../README.md#validate)
```

Comment:
The same as for README, should be "Validate installation"

**Note:** If we preserve this line in README we should do not forget to delete it from here